Sources/TensorFlow/Operators/NN.swift

```diff
@@ -353,6 +430,24 @@ func _vjpdepthwiseConv2dBackpropInput<Scalar: TensorFlowFloatingPoint>(
     })
 }
 
+/// TensorFlow builtin depthwiseConv2D gradient helper for the filter.
+@differentiable(wrt: (x, input), vjp: _vjpdepthwiseConv2dBackpropFilter)
```

Comment:
```suggestion
@differentiable(wrt: (x, input), vjp: _vjpDepthwiseConv2dBackpropFilter)
```

Reply:
Hey @eaplatanios. Thanks for helping me out. I'm not sure what to change here? Could you please clarifiy.

Reply:
@eaplatanios Good eye! I didn't realize that wasn't camel case. I wasn't paying too much attention to naming when moving things around. Thanks!